### PR TITLE
Add new model ID for Saswell TRVs

### PR DESCRIPTION
**Pull request by drzony**

### CODE REVIEW

**Comment by hnykda** [comment]
Really appreciated, was just doing it by myself.

**Comment by drzony** [comment]
I don't know what the release process is, so there is a possibility that the above change was not built yet.

**Comment by hnykda** [comment]
Well, according to https://github.com/Koenkk/zigbee-herdsman-converters/commits/master, there has been 14.0.150 release of this repo. In https://github.com/Koenkk/zigbee2mqtt/commits/dev, I can see updating to this version: https://github.com/Koenkk/zigbee2mqtt/commit/7bdf52d824a6a2c37027f42f8e1593f59e3a21a5 . But when I try it again, it doesn't work.

**Comment by drzony** [comment]
In your log I've seen commit #45aa352, so this means that your version did not contain the fix

**Comment by hnykda** [comment]
Sorry for confusion - I have tried again with 7bdf52d and that's what I was referring to:

```
info  2021-05-19 09:25:49: Logging to console and directory: '/config/zigbee2mqtt/log/2021-05-19.09-25-42' filename: log.txt
info  2021-05-19 09:25:49: Starting Zigbee2MQTT version 1.18.3-dev (commit #7bdf52d)
info  2021-05-19 09:25:49: Starting zigbee-herdsman (0.13.102)
info  2021-05-19 09:25:53: zigbee-herdsman started
info  2021-05-19 09:25:53: Coordinator firmware version: '{"meta":{"maintrel":3,"majorrel":2,"minorrel":6,"product":0,"revision":20201127,"transportrev":2},"type":"zStack12"}'
info  2021-05-19 09:25:53: Currently 10 devices are joined:
info  2021-05-19 09:25:53: TestZarovka1 (0x00158d0005726289): ZNLDP12LM - Xiaomi Aqara smart LED bulb (Router)
info  2021-05-19 09:25:53: SenzorVlhkostiATeploty (0x00158d00068b6b01): WSDCGQ11LM - Xiaomi Aqara temperature, humidity and pressure sensor (EndDevice)
info  2021-05-19 09:25:53: PohybovySenzor (0x00158d0006ba9490): RTCGQ11LM - Xiaomi Aqara human body movement and illuminance sensor (EndDevice)
info  2021-05-19 09:25:53: AqaraCube (0x00158d0005cf3268): MFKZQ01LM - Xiaomi Mi/Aqara smart home cube (EndDevice)
info  2021-05-19 09:25:53: DoubleSwitch (0x00158d0006515e80): WXKG07LM - Xiaomi Aqara D1 double key wireless wall switch (EndDevice)
info  2021-05-19 09:25:53: TripleSwitch (0x04cf8cdf3c7d8f7a): WXCJKG13LM - Xiaomi Aqara Opple switch 3 bands (EndDevice)
info  2021-05-19 09:25:53: HlaviceElza (0xbc33acfffe05907f): TS0601_thermostat - TuYa Radiator valve with thermostat (EndDevice)
info  2021-05-19 09:25:53: GledoptoGeorge (0x00124b0022612a0d): GL-C-008-2ID - Gledopto Zigbee LED Controller RGB+CCT (2 ID) (Router)
info  2021-05-19 09:25:53: TRVLivingRoom (0x804b50fffe4466e0): SEA801-Zigbee/SEA802-Zigbee - Saswell Thermostatic radiator valve (EndDevice)
info  2021-05-19 09:25:53: 0x804b50fffe446a4c (0x804b50fffe446a4c): Not supported (EndDevice)
info  2021-05-19 09:25:53: Zigbee: disabling joining new devices.
info  2021-05-19 09:25:53: Started frontend on port 0.0.0.0:8099
error 2021-05-19 09:25:53: Not connected to MQTT server!
error 2021-05-19 09:25:53: Cannot send message: topic: 'zigbee2mqtt/bridge/event', payload: '{"data":{"friendly_name":"0x804b50fffe446a4c","ieee_address":"0x804b50fffe446a4c"},"type":"device_announce"}
error 2021-05-19 09:25:53: Not connected to MQTT server!
error 2021-05-19 09:25:53: Cannot send message: topic: 'zigbee2mqtt/bridge/log', payload: '{"message":"announce","meta":{"friendly_name":"0x804b50fffe446a4c"},"type":"device_announced"}
info  2021-05-19 09:25:53: Connecting to MQTT server at mqtt://core-mosquitto:1883
warn  2021-05-19 09:25:54: Received message from unsupported device with Zigbee model 'TS0601' and manufacturer name '_TZE200_zuhszj9s'
warn  2021-05-19 09:25:54: Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html.
info  2021-05-19 09:25:55: Connected to MQTT server
```

**Comment by drzony** [comment]
Have you tried re-pairing the TRVs?

**Comment by hnykda** [comment]
Not yet, will do as I wrote in https://github.com/Koenkk/zigbee-herdsman-converters/pull/2594#issuecomment-843830833 (physically not on site atm). Will get back in a few hours how it went

**Comment by hnykda** [comment]
So I tried the mentioned [commit (log)](https://gist.github.com/hnykda/87e4610e41800fc120a0eeb445d2bfc3#file-gistfile1-txt-L2).

1. [I removed the device](https://gist.github.com/hnykda/87e4610e41800fc120a0eeb445d2bfc3#file-gistfile1-txt-L129-L130)
2. it pairs fine, but throws an error [about not being supported](https://gist.github.com/hnykda/87e4610e41800fc120a0eeb445d2bfc3#file-gistfile1-txt-L153)

**Comment by hnykda** [comment]
image

File: devices/saswell.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ module.exports = [
             {modelID: 'TS0601', manufacturerName: '_TZE200_c88teujp'},
             {modelID: 'TS0601', manufacturerName: '_TZE200_yw7cahqs'},
             {modelId: 'TS0601', manufacturerName: '_TZE200_azqp6ssj'},
+            {modelID: 'TS0601', manufacturerName: '_TYST11_zuhszj9s'},
```
Comment (hnykda) [inline devices/saswell.js:18]: @drzony Isn't this typo in the end :hushed: ? In my log, I see `_TZE200_zuhszj9s`, not `_TYST11_zuhszj9s`.

Comment (hnykda) [reply devices/saswell.js:18]: tried here maybe? https://github.com/Koenkk/zigbee-herdsman-converters/pull/2601

Comment (drzony) [reply devices/saswell.js:18]: Yes, that's the problem. Copy-Paste error on my side

Comment (hnykda) [reply devices/saswell.js:18]: Thanks, really appreciate the help and the pointers :+1:

**Pull request merged**